app/models/manageiq/providers/openstack/network_manager.rb

```diff
@@ -147,7 +146,7 @@ def create_floating_ip_queue(userid, options = {})
     }
     queue_opts = {
       :class_name  => self.class.name,
-      :method_name => 'create_floating_ip',
+      :method_name => 'create_security_group',
```

Comment:
BTW all those methods are going to replaced by a builder: https://github.com/ManageIQ/manageiq/pull/13074